BUILD.bazel

```diff
@@ -479,29 +479,6 @@ cc_library(
     ],
 )
 
-# This header is used to warp some internal code so we can reduce suspicious
-# symbols export.
-cc_library(
-    name = "exported_internal",
-    srcs = glob(
-        [
-            "src/ray/internal/internal.cc",
```

Comment:
Guess it is unrelated changes? Looks okay to include in this PR to me.

Reply:
Actually it is... There is some method used in this file got impacted by this PR. So when I compile it just failed :( 
I notice it seems not useful, so just delete that file.